config/webpack/plugin/SetupPlugin.js

```diff
@@ -65,17 +65,31 @@ class SetupPlugin {
       test: /\.md$/,
       oneOf: [
         {
-          // To preserve previous behaviour, we will only apply mdx to requests issued
-          // from either the content config files or from .md or .mdx files
+          // Use MDX to import any md files imported from an mdx file.
           issuer: [
-            /dev-site-config.*\/contentConfig\.js$/,
             /\.mdx?$/,
```

Comment:
should this be `/\.mdx$/,`

Reply:
no, this is correct if you have a .mdx file that imports a .md file that imports a .md file you'd want it to go through this loader.

standard .md files can't import so they won't hit this check.